### PR TITLE
AMQP confirms/rejects

### PR DESCRIPTION
**Pull request by jupiter**
#48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,6 +621,22 @@ Returns a consumer listening on this topic. Either existing or new. Correspondin
...
 Stops listening for messages on this topic. If `config.cleanupConsumers` is set, and this consumer was created using `channelManager.findOrCreateConsumer`, it would be removed from the `channelManager`.
 
+#### consumer.onceConsuming(cb)
+
+- **cb** Function that is called if the consumer is already consuming or once when it next starts consuming.
+
+#### consumer.confirmProcessedMessage(message)
+
+Confirms with the broker (where supported) that processing of this message has completed. Only works where `config.autoConfirm` is set to `false`.
+
```
Comment (HCanber) [inline README.md:631]: Maybe add a sentence what this actually means for Redis and AMQP adapters.

File: README.md [hunk 2]
```diff
@@ -620,6 +621,22 @@ Returns a consumer listening on this topic. Either existing or new. Correspondin
...
+- **message** The message originally emitted by this consumer, by reference.
+
+#### consumer.rejectMessage(message)
+
+Confirms with the broker (where supported) that this message should not be processed, e.g. in cases such as invalid message or TTL reached. Only works where `config.autoConfirm` is set to `false`.
+
```
Comment (HCanber) [inline README.md:637]: Maybe add a sentence what this actually means for Redis and AMQP adapters.

**Comment by jupiter** [comment]
@HCanber Would you please merge this for me?

**Comment by HCanber** [comment]
@jupiter Sorry for not merging. For some reason I'm not getting email notifications so I missed this.

**Pull request merged**